code/modules/paperwork/paper.dm

```diff
@@ -484,3 +484,66 @@
 	qdel(src)
+
+/obj/item/paper/telecomms
+	name = "Your Guide To Telecomms!"
+	info = "<b>Congratulations On Your Purchase Of An NT-07 Telecommunications System!</b><br>\
+	WARNING: Should this machinery become destroyed or non functional, please contact your nearest technician.<br>\
+	INFO: This machinery is very subsceptable to Ionspheric Interference. Should an Ionspheric Anomalie occur, the machinery may be offline for a short amount of time.<br>\
+	<br>\
+	List of authorised channels and corresponding frequencies:<br>\
+	<li>Science: 135.1</li>\
+	<li>Medical: 135.5</li>\
+	<li>Supply: 134.7</li>\
+	<li>Service: 134.9</li>\
+	<li>Command: 135.3</li>\
+	<li>Security: 135.9</li>\
+	<li>Engineering: 135.7</li>\
+	<li>Common: 145.7</li><br>\
+	<br>\
+	This Topology:<br>\
+	This topology is split into isles, each isle having 1 bus and 1 CPU responsible for 2 channels, each with their own servers. The channel specific servers are marked with floor tiles similar to each channel identifier.<br>\
+	<br>\
+	Telecommunication Machinery Description<br>\
+	All machines share similar features, such as networks and filtering frequencies, but each machine has a very specific role<br>\
+	Servers:<br>\
+	Servers store a data packet every time audio is sent down a corresponding channel, this data can be viewed with a Telecomms Server Montior. Servers can also be used for scripting, but sadly this is no longer supported.<br>\
+	<br>\
+	Hubs:<br>\
+	Hubs are the 'Hub' of the setup, quite literally. All the machine except processors (see below) link to this, including distant ones such as relays (See below).<br>\
+	<br>\
+	Processors:<br>\
+	Processors are an essential part of a telecomms setup if you want to be able to understand anything. They take an encoded signal, and decode it into standard audio which is sent off to the broadcasters (See below). These link to the bus mainframes.<br>\
+	<br>\
+	Bus Mainframes<br>\
+	Bus Mainframes play a similar role to hubs. Networks without hubs will use a Bus Mainframe as a hub, as they can do local machinery but not relays. The bus links to all machines that cover the channels it manages. So to the hub, the processor and the server.<br>\
+	<br>\
+	Reciever:<br>\
+	Recievers are what they say on the tin, they recieve subspace signals. If you are using a hub, link it to the hub, if you are not, link it to the mainframe.<br>\
+	<br>\
+	Broadcasters:<br>\
+	Broadcasters are what they say on the tin, they take decoded signals from the processors, and broadcast them through subspace. If you are using a hub, link it to the hub, if you are not, link it to the mainframe.<br>\
+	<br>\
+	Other Machinery:<br>\
+	This machinery is not linked directly to the network, but is still important for communications.<br>\
+	<br>\
+	PDA Server:<br>\
+	This is the server responsible for PDA messaging, if this is disabled, PDA messaging will fail to work, and thus not work.<br>\
```

Comment:
lol

` if this is disabled, PDA messaging will fail to work, and thus not work.` -> ` if this is disabled, PDA messaging will not function.`